fastlane/docs/Actions.md

```diff
@@ -886,7 +886,7 @@ See how [Artsy](https://github.com/fastlane/examples/blob/master/Artsy/eidolon/F
 ### [Crashlytics Beta](http://try.crashlytics.com/beta/)
 ```ruby
 crashlytics(
-  crashlytics_path: './Crashlytics.framework', # path to your 'Crashlytics.framework'
+  crashlytics_path: './Pods/Crashlytics/', # path to your Crashlytics submit binary.
```

Comment:
You mean `./Pods/iOS/Crashlytics/`. Perhaps with a different cocoapods version than I am using. The point, though, is to make clear that the location of `submit` is important and not `Crashlytics.framework`.